site/community/release_guide.md

```diff
@@ -0,0 +1,475 @@
+
+Once all issues have been resolved, you should go back and build a new release candidate with these changes.
+
+### Checklist to proceed to the next step
+
+1. Issues identified during vote have been resolved, with fixes committed to the release branch.
+
+**********
+
+## Finalize the release
+
+Once the release candidate has been reviewed and approved by the community, the release should be finalized. This involves the final deployment of the release candidate to the release repositories, merging of the website changes, etc.
+
+### Deploy artifacts to Maven Central Repository
+
+Use the Apache Nexus repository to release the staged binary artifacts to the Maven Central repository. In the `Staging Repositories` section, find the relevant release candidate `orgapachebookkeeper-XXX` entry and click `Release`. Drop all other release candidates that are not being released.
+
+#### Deploy source release to dist.apache.org
+
+Copy the source release from the `dev` repository to the `release` repository at `dist.apache.org` using Subversion.
+
+    svn move https://dist.apache.org/repos/dist/dev/bookkeeper/bookkeeper-${VERSION}-rc${RC_NUM} https://dist.apache.org/repos/dist/release/bookkeeper/bookkeeper-${VERSION}
+
+### Update Website
+
+[TBD]
+
+### Mark the version as released in JIRA and Github
+
+In JIRA, inside [version management](https://issues.apache.org/jira/plugins/servlet/project-config/BOOKKEEPER/versions), hover over the current release and a settings menu will appear. Click `Release`, and select today’s date.
+
+In Github, inside [milestones](https://github.com/apache/bookkeeper/milestones), hover over the current milestone and click `close` button to close a milestone and set today's date as due-date.
+
+### Checklist to proceed to the next step
+
+* Maven artifacts released and indexed in the [Maven Central Repository](https://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.apache.bookkeeper%22)
+* Source and Binary distribution available in the release repository of [dist.apache.org](https://dist.apache.org/repos/dist/release/incubator/bookkeeper/)
+* Website is updated with new release
+* Release tagged in the source code repository
+* Release version finalized in JIRA and Github
+
+**********
+
+## Promote the release
+
+Once the release has been finalized, the last step of the process is to promote the release within the project and beyond.
+
+### Apache mailing lists
+
+Announce on the dev@ mailing list that the release has been finished.
+
+Announce on the release on the user@ mailing list, listing major improvements and contributions.
+
+(NOTE: Make sure sending the announce email using apache email, otherwise announce@apache.org will reject your email.)
+
+
+    From: xxx@apache.org
+    To: dev@bookkeeper.apache.org, dev@bookkeeper.apache.org, announce@apache.org
+    Subject: [ANNOUNCE] Apache BookKeeper x.y.z released
+     
```

Comment:
Include description of the project and why people would be interested in this release.